README.md

```diff
@@ -18,8 +18,9 @@ Add [`rc`] to your autoload or source [`connect.kak`] and its [modules] manually
 ### Add to desktop
 
 1. Copy [`kakoune-connect.desktop`] to `$XDG_DATA_HOME/applications`.
-2. Add [`kak-connect`] to your path.
-3. Open `$XDG_CONFIG_HOME/mimeapps.list` and add the following MIME type association:
+2. Copy Kakoune's logo (`https://github.com/mawww/kakoune/blob/master/doc/kakoune_logo.svg`) to `$XDG_DATA_HOME/icons/hicolor/scalable/apps/kakoune.svg`.
```

Comment:
It is not a reference link.

Reply:
Sorry, I don't know what you mean. You can write it yourself as you want it.

Reply:
``` markdown
2. Copy [Kakoune’s logo] to `$XDG_DATA_HOME/icons/hicolor/scalable/apps/kakoune.svg`.

[Kakoune’s logo]: https://github.com/mawww/kakoune/blob/master/doc/kakoune_logo.svg
```